Cargo.toml

```diff
@@ -42,20 +42,21 @@ serde_derive = "1.0.136"
 colored = "2.0.0"
 itertools = "0.10.3"
 regex = "1.5.5"
-jwalk = "0.6.0"
-clap = { version = "3.1.12", features = ["derive"] }
+jwalk = "0.8.1"
+clap = { version = "4.0.3", features = ["derive"] }
 log = "0.4.16"
-env_logger = "0.9.0"
+env_logger = "0.10.0"
 tempdir = "0.3"
 serde_json = "1.0.82"
-tree-sitter-kotlin = { git = "https://github.com/ketkarameya/tree-sitter-kotlin.git" }
-tree-sitter-java = { git = "https://github.com/tree-sitter/tree-sitter-java.git" }
-tree-sitter-swift = { git = "https://github.com/ketkarameya/tree-sitter-swift.git", branch = "add_parser" }
-tree-sitter-python = { git = "https://github.com/tree-sitter/tree-sitter-python.git" }
-tree-sitter-typescript = { git = "https://github.com/tree-sitter/tree-sitter-typescript.git" }
-# TODO: tree-sitter-go should be changed either to upstream or Uber main branch
+# TODO: Update after https://github.com/fwcd/tree-sitter-kotlin/pull/71 lands
+tree-sitter-kotlin = { git = "https://github.com/ketkarameya/tree-sitter-kotlin.git", branch = "expose_node_function_value_parameters" }
```

Comment:
Are these imports just pulling the latest commit in that branch? That's kind of horrifying from the point of view of reproducible builds. Is there a way to lock the remaining ones to a specific commit, rather than the head of a branch?

Either way, happy we are doing less of that...

Reply:
Addressed https://github.com/uber/piranha/pull/315/commits/418ebde9f182a78fbe6113f9dade6a06f07d2857